src/models/Events.ts

```diff
@@ -0,0 +1,33 @@
+export interface MatrixRoomEvent {
```

Comment:
Nothing in this file is how I want to define events in this SDK - please leave this as a task for https://github.com/turt2live/matrix-js-bot-sdk/issues/11 and similar issues.

Reply:
It would be much easier if you actually defined how you'd want them to look.

Reply:
correct, but I don't really know what that is yet ._.

Just "not this" unfortunately.